dash/dash.py

```diff
@@ -325,6 +335,40 @@ def react(self, *args, **kwargs):
             'Use `callback` instead. `callback` has a new syntax too, '
             'so make sure to call `help(app.callback)` to learn more.')
 
+    def _generate_footer_or_header_html(self, footer_or_header):
+        section = getattr(self, footer_or_header)
+        if isinstance(section, collections.Callable):
+            section_instance = section()
+        else:
+            section_instance = section
+
+        def convert_to_html(element):
+            if isinstance(element, Component):
+                element_html = element.to_html5()
+            elif isinstance(element, basestring):
```

Comment:
failing in python3. need:
`from past.builtins import basestring    # pip install future`